src/sdam/monitor.ts

```diff
@@ -207,27 +223,37 @@ function resetMonitorState(monitor: Monitor) {
   monitor[kConnection] = undefined;
 }
 
+function useStreamingProtocol(monitor: Monitor, topologyVersion: TopologyVersion | null): boolean {
+  // If we have no topology version we always poll no matter
+  // what the user provided.
+  if (topologyVersion == null) return false;
```

Comment:
For clarification, in this case, the server doesn't support streaming at all right?

Reply:
Yes, I've updated the comment to reflect that.